### DISCUSSION TITLE
Module in Tab

### DISCUSSION THREAD

**Discussion by ghost** [serenity-is/Serenity#6438] [Help]
does anyone know how to load a module in a new tab?

**Comment by edwardch** [comment]
Try add `Target = "_blank"` to the navigationlink?

**Reply by stixoffire** [reply]
If the tab is there , then you want the ~_TabMyModule  to be in your template and use that when you have your function 
So the container code that uses getTemplate should have it where you have something like this <div id="~_TabMyModule "> </div>
That is a placeholder and gets replaced .. when you then do something like this.. NOTICE the ~_ is not used here in the object  creation.

```

myModule = new module($('TabMyModule') ;
myModule.open()
```
